WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerActivity.java

```diff
@@ -565,4 +580,25 @@ public void onClick(DialogInterface dialog, int which) {
             return builder.create();
         }
     }
+
+    private void startEditingVisibility() {
+        mRecycleView.setItemAnimator(new DefaultItemAnimator());
+        getAdapter().setEnableEditMode(true);
+        startSupportActionMode(new ActionModeCallback());
+    }
+
+    private void showRemoveSelfHostedSiteDialog(@NonNull final SiteModel site) {
+        AlertDialog.Builder dialogBuilder = new AlertDialog.Builder(this);
+        dialogBuilder.setTitle(getResources().getText(R.string.remove_account));
+        dialogBuilder.setMessage(getResources().getText(R.string.sure_to_remove_account));
+        dialogBuilder.setPositiveButton(getResources().getText(R.string.yes), new DialogInterface.OnClickListener() {
+            public void onClick(DialogInterface dialog, int whichButton) {
+                mDispatcher.dispatch(SiteActionBuilder.newRemoveSiteAction(site));
+                getAdapter().loadSites();
```

Comment:
This probably should be inside of `public void onSiteRemoved(OnSiteRemoved event) {` instead of here since the remove site action is async.